lib/kafo/kafo_configure.rb

```diff
@@ -446,6 +419,7 @@ def run_installation
         log_parser = PuppetLogParser.new
         logger = Logger.new('configure')
 
+        require 'pty'
```

Comment:
I dislike these mid-code requires. They can hide errors, and lead to missing libraries at runtime.